bayesian/utility/db_gateway.py

```diff
@@ -56,6 +58,14 @@ class GraphAnalyses:
             .valueMap().fold()).fill(epv);};epv;
             """
 
+    get_vuln_query = """
+            g.V().has('snyk_ecosystem', ecosystem).has('package_name', within(packages)).valueMap()
+            """
+
+    get_package_query = """
+            g.V().has('ecosystem', ecosystem).has('name', within(packages)).valueMap()
```

Comment:
Don't we want to query `synk_ecosystem` node..?

Reply:
This is a query for the package node not for vulnerability node. @deepak1725

Reply:
We query `snyk_ecosystem` node only for getting vulnerabilities, this query is to get package details for vulnerable packages.